src/Shared/SchemaUtils.cs

```diff
@@ -5,5 +5,5 @@ namespace Aspire.Hosting.Utils;
 
 internal static class SchemaUtils
 {
-    public const string SchemaVersion = "https://json.schemastore.org/aspire-8.0.json";
+    public const string SchemaVersion = "https://json.schemastore.org/aspire-9.0.json";
```

Comment:
Should this go into the release/9.0 branch in our first servicing?

Reply:
Need to discuss strategy here with @davidfowl, @vhvb1989, and @ellismg

Reply:
It's unfortunate that using the 9.0.0 bits today, can produce a manifest that doesn't conform to the schema it says it conforms to. For example:

1. Call the `PublishAsAzureContainerApp` method
2. This will produce a manifest that points to `https://json.schemastore.org/aspire-8.0.json` and also has `project.v1`, which isn't valid for that schema version.

Our playground app does this today:

https://github.com/dotnet/aspire/blob/11c51e7a34d068eb1881ee69e2dd3668511718d2/playground/AzureContainerApps/AzureContainerApps.AppHost/aspire-manifest.json#L2

https://github.com/dotnet/aspire/blob/11c51e7a34d068eb1881ee69e2dd3668511718d2/playground/AzureContainerApps/AzureContainerApps.AppHost/aspire-manifest.json#L105-L106

Reply:
Yeah so there are two approaches we can take here. We can update the -8.0 schema on schema store with the one we have in our repo and just not bump the version, or we can ship a -9.0 schema and make sure it works everywhere.

Reply:
I'd assume bumping the version is the "right" thing to do. It doesn't really make sense to add things and not change the version. Because what's the point of versioning it then?

Reply:
It can be done dynamically